### ISSUE TITLE
Baseline WARNINGS/ERROR in master 

### ISSUE THREAD

**Issue by kwin** [Adobe-Consulting-Services/acs-aem-commons#2169]
Since version 4.4.0 is available from the Adobe Maven Repo  baselining in latest master fails with one error and several warnings:

```
[ERROR] com.adobe.acs.commons.mcp.model: Version increase required; detected 4.2.0, suggested 4.3.0
[WARNING] com.adobe.acs.commons.cloudconfig: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.cloudconfig: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.filefetch: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.filefetch: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo.client: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo.client: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.mcp.form.workflow: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.mcp.form.workflow: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[INFO] Baseline analysis complete, 1 error(s), 10 warning(s)
```
The error is a regression of https://github.com/Adobe-Consulting-Services/acs-aem-commons/commit/ee1674c994d4eb9b61e8596ededf82f87299bddd
The warnings are due to the fact that those packages are not versioned explicitly and therefore get the version of the bundle. Every exported package should get a dedicated package version!

**Comment by kwin** [comment]
@joerghoh Can you look into this, as you are the author of https://github.com/Adobe-Consulting-Services/acs-aem-commons/commit/ee1674c994d4eb9b61e8596ededf82f87299bddd?

**Comment by joerghoh** [comment]
Will do

**Comment by joerghoh** [comment]
@kwin please check #2170

**Comment by kwin** [comment]
The error is fixed with #2170. Still the warnings should be fixed as well

**Comment by joerghoh** [comment]
@paul-bjorkstrand  fixed a number of these warnings with 0dc233a36271b9308343374ce48cebd89605161f

**Comment by joerghoh** [comment]
I raised #2179 to address the remaining baseline warning.
